.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      linux_exclude_swift_versions: '[{"swift_version": "5.9"}, {"swift_version": "5.10"}]'
+      linux_pre_build_command: apt-get update && apt-get install -y libsqlite3-dev libncurses5-dev
+      windows_exclude_swift_versions: '[{"swift_version": "5.9"}, {"swift_version": "5.10"}]'
```

Comment:
We might just want `macos_xcode_versions: '["16.3"]'` as I'm not sure there's value in testing on earlier macOS versions. @owenv, what do you think?

Reply:
yes, I think we can restrict this to just 16.3

Reply:
macOS support will be added in the future.